roles/swift-account/tasks/main.yml

```diff
@@ -13,6 +13,14 @@
             dest=/etc/swift/account-server.conf owner=swift group=swift
   notify: restart swift-account service
 
+- name: trigger restart on upgrades
```

Comment:
Have you tested in a Swift world? Each service type appears to have it's own restart services set, that are triggered for config changes and such. The handlers live in each swift role. If that is missing services, the fix is to correct those handlers, not call a different handler.